docs/native/src/faq/filetypes.rst

```diff
@@ -0,0 +1,16 @@
+What file types are supported in |Office iOS|?
```

Comment:
I looked for these as well, but I could only find articles for Word & Excel:
- Word: https://support.office.com/en-us/article/Read-only-and-compatible-files-3c7ee711-27de-4cd5-8125-a6e1f5ed30c4?ui=en-US&rs=en-US&ad=US
-Excel: https://support.office.com/en-us/article/Read-only-and-compatible-files-2daa6bd6-8db5-4521-9f37-7e23b3e4bdab?ui=en-US&rs=en-US&ad=US

There was also an article about using the conversion service which may be a good substitute: https://support.office.com/en-us/article/Why-does-the-Microsoft-online-service-need-to-convert-some-Office-files-ef8ff042-c40c-4d9f-9d11-0d21f88e4179?ui=en-US&rs=en-US&ad=US